pkg/image/airgap.go

```diff
@@ -121,27 +121,29 @@ func CopyAirgapImages(opts imagetypes.ProcessImageOptions, log *logger.CLILogger
 		LogForUI:       true,
 	}
 
+	result := &imagetypes.CopyAirgapImagesResult{}
 	if opts.AirgapBundle != "" {
```

Comment:
can we just move this to the top of the function and return if it's not set? would be a bit cleaner.